meson_options.txt

```diff
@@ -0,0 +1,14 @@
+# Feature options
+option('tests', type : 'feature', value : 'auto', yield : true, description : 'Build tests')
+option('examples', type : 'feature', value : 'auto', yield : true, description : 'Build examples')
+option('enable-gtk-doc', type : 'boolean', value : false, description : 'Use gtk-doc to build documentation')
```

Comment:
Why is `gtk-doc` the only one with `enable`? Won't this create something like `enable-enable-gtk-doc`?

Reply:
Or, the other way around: leaving an ambiguous `--tests` option

Reply:
The options `tests` and `examples` were ambiguous, now the options are:

-   enable-examples
-   enable-gtk-doc
-   enable-tests
-   package-name
-   package-origin
-   with-gstd-logstatedir
-   with-gstd-runstatedir